Amazon.QLDB.Driver/transaction/AsyncTransaction.cs

```diff
@@ -55,7 +55,7 @@ internal AsyncTransaction(Session session, string txnId, ILogger logger, Cancell
         /// </summary>
         ///
         /// <returns>A task representing the asynchronous abort operation.</returns>
-        internal async Task Abort()
+        public async Task Abort()
```

Comment:
Does it need to be public?

Reply:
Yes, in order for this class to implement `IAsyncTransaction` interface, the override methods need to be public.